app/src/main/java/swen_anigans/mathematicfanatic/ClassroomActivity.java

```diff
@@ -168,21 +150,18 @@ public void goToLocker(View view){
     }
 
     public void checkAnswer(View view){
-        EditText answerInput = (EditText) findViewById(R.id.editClassroomAnswer);
-        String answer = answerInput.getText().toString();
-        CharSequence text;
-        if (!answer.isEmpty()) {
-            int quizAnswer = Integer.parseInt(answer);
-            answers.set(pageNumber-1, quizAnswer);
-            boolean correct;
-            if(quizAnswer == expectedAnswers.get(pageNumber - 1)){
-                text = "Correct!";
+        TextView answerInput = (TextView) findViewById(R.id.editClassroomAnswer);
+        String answerText = answerInput.getText().toString();
+        String text;
+        if(!answerText.isEmpty()){
```

Comment:
there really isn't a point.  right now save answer is called when either of the arrow buttons are pressed. if save answer was added here saveanswer would be called again when the arrow was pressed. i didn't really see it as being necessary